docs/.sphinx/_toc.yml.in

```diff
@@ -0,0 +1,24 @@
+# Anywhere {branch} is used, the branch name will be substituted.
+# These comments will also be removed.
+defaults:
+  numbered: False
+root: index
+subtrees:
+  - entries:
+    - file: hipify-clang
+    - file: hipify-perl
+    - file: supported_apis
+      subtrees: 
+        - entries:
+          - file: tables/CUDA_Runtime_API_functions_supported_by_HIP
+          - file: tables/CUDA_Driver_API_functions_supported_by_HIP
+          - file: tables/cuComplex_API_supported_by_HIP
+          - file: tables/CUDA_Device_API_supported_by_HIP
+          - file: tables/CUDA_RTC_API_supported_by_HIP
+          - file: tables/CUBLAS_API_supported_by_HIP
+          - file: tables/CURAND_API_supported_by_HIP
+          - file: tables/CUDNN_API_supported_by_HIP
+          - file: tables/CUFFT_API_supported_by_HIP
+          - file: tables/CUSPARSE_API_supported_by_HIP
+          - file: tables/CUB_API_supported_by_HIP
```

Comment:
Should it be supported explicitly?

Reply:
**[ToFix]**
The paths to CUDA2HIP docs are changed, so the corresponding paths in the links of `README.md`  (#-supported-cuda-apis) have to be changed as well.

Reply:
Fixed in [c684bf8](https://github.com/ROCm-Developer-Tools/HIPIFY/pull/737/commits/c684bf8b46240827e5246399513416694802ca45)